static/styles.css

```diff
@@ -269,44 +269,39 @@ pre code {
 
 #editor-container {
     height: 100%;
+    display: flex;
+    flex-direction: row;
     top: 32px;
 }
 
 .left {
-    width: calc(100vw - 640px);
-    height: calc(99vh - calc(36px + 32px));
+    width: 100%;
+    display: flex;
+    flex-direction: column;
+    height: calc(100vh - 42px);
     background-color: #000;
-    padding: 0px;
-    vertical-align: top;
 }
 
 .right {
-    width: 640px;
-    /* 640 x 360 */
-    min-width: 240;
-    /* 240 x 135 */
-    max-width: 960;
-    /* 960 x 540 */
-    max-height: calc(99vh - calc(32px + 32px));
+    width: 50vw;
```

Comment:
If I set the width of the lesson guide to 640px, it will cause erratic behavior in the tabs section. To fix that, I will need to adjust the media query which might cause laptop users to see the mobile version of the page